Links in den docs auf die Umgebung anpassen

Vor allem in `Configure.md` haben wir Links, die sind hardgecodet mit `localhost:8080` drin, das würde ich gerne durch die eigentlichen Links der Umgebungen ersetzen.

Das habe ich auch schon angefangen einzubauen, allerdings kommt es in der Pipeline (nicht erst seitdem) zu dem Fehler 

```
#67 1.010 [ERROR] Error: "***" does not look like a valid URL. Make sure it has a protocol; for example, "https://example.com/".
#67 1.010 
#67 1.010     at validateConfig (/build/fpo/node_modules/@docusaurus/core/lib/server/configValidation.js:396:15)
#67 1.010     at loadSiteConfig (/build/fpo/node_modules/@docusaurus/core/lib/server/config.js:40:62)
#67 1.010     at async Promise.all (index 1)
#67 1.010     at async loadContext (/build/fpo/node_modules/@docusaurus/core/lib/server/site.js:39:97)
#67 1.010     at async getLocalesToBuild (/build/fpo/node_modules/@docusaurus/core/lib/commands/build/build.js:55:21)
#67 1.010     at async Command.build (/build/fpo/node_modules/@docusaurus/core/lib/commands/build/build.js:30:21)
#67 1.010     at async Promise.all (index 0)
#67 1.010     at async runCLI (/build/fpo/node_modules/@docusaurus/core/lib/commands/cli.js:56:5)
#67 1.010     at async file:///build/fpo/node_modules/@docusaurus/core/bin/docusaurus.mjs:44:3
#67 1.011 [INFO] Docusaurus version: 3.9.1
```

Lokal tritt das nicht auf, müsste man mal Zeit investieren wieso das so ist.

--- a/Dockerfile
+++ b/Dockerfile
@@ -28,14 +28,12 @@ COPY scripts/pdf_docs.py /scripts/pdf_docs.py
 
 WORKDIR /build/fpo
 
+# Replace localhost links with SERVER_HOST
+RUN find . -name "*.md" -type f -exec sed -i "s|http://localhost:8080|${SERVER_HOST}|g" {} \;
+
 # Install dependencies & build
 RUN npm install
 RUN npm run build
-
-# Replace localhost links with SERVER_HOST
-RUN find . -name "*.md" -type f -exec sed -i "s|http://localhost:8080|${SERVER_HOST}|g" {} \;
-RUN find . -name "*.html" -type f -exec sed -i "s|http://localhost:8080|${SERVER_HOST}|g" {} \;
-RUN find . -name "*.js" -type f -exec sed -i "s|http://localhost:8080|${SERVER_HOST}|g" {} \;
 
 WORKDIR /build/fpo/docs
 

--- a/docs/2-configure.md
+++ b/docs/2-configure.md
@@ -18,18 +18,18 @@ override to have SSL properly set up, see the appropriate section below
 
 ## Overview
 
-- the frontend user interface is available at `/`, e.g., [localhost:8080](http://localhost:8080/) for local development
-- the backend api is available at `/api/`, e.g., [localhost:8080/api/](http://localhost:8080/api/) for local development
+- the frontend user interface is available at `/`, e.g., [/](http://localhost:8080/)
+- the backend api is available at `/api/`, e.g., [/api/](http://localhost:8080/api/)
 - internally, the services are reachable by their respective hostnames:
     - Postgres: `postgres`
     - Backend: `api`
-- a collection of development tools and links is available at `/dev/`, e.g., [localhost:8080/dev/](http://localhost:8080/dev/)
-    - a simple database client is available at `/pgweb/`, e.g., [localhost:8080/pgweb/](http://localhost:8080/pgweb/) for local development. For deployment, this should be password protected, see [Environment](#Environment)
-    - the API documentation is available at `/swagger/`, e.g., [localhost:8080/swagger](http://localhost:8080/swagger/)
-    - the documentation hosted in its entirety at `/docs/`, e.g. [localhost:8080/docs/](http://localhost:8080/docs/)
-        - the hosted backend documentation is available at `/dev/haddock/`, e.g. [localhost:8080/dev/haddock/](http://localhost:8080/dev/haddock/)
-        - the hosted frontend documentation is available at `/dev/purs/`, e.g. [localhost:8080/dev/purs](http://localhost:8080/dev/purs/)
-    - backend logs can be found at `dev/logs.html`, e.g. [localhost:8080/dev/logs.html](http://localhost:8080/dev/logs.html). You can login here with any credentials that are also superadmins within the application.
+- a collection of development tools and links is available at `/dev/`, e.g., [/dev/](http://localhost:8080/dev/)
+    - a simple database client is available at `/pgweb/`, e.g., [/pgweb/](http://localhost:8080/pgweb/). For deployment, this should be password protected, see [Environment](#Environment)
+    - the API documentation is available at `/swagger/`, e.g., [/swagger/](http://localhost:8080/swagger/)
+    - the documentation hosted in its entirety at `/docs/`, e.g. [/docs/](http://localhost:8080/docs/)
+        - the hosted backend documentation is available at `/dev/haddock/`, e.g. [/dev/haddock/](http://localhost:8080/dev/haddock/)
+        - the hosted frontend documentation is available at `/dev/purs/`, e.g. [/dev/purs/](http://localhost:8080/dev/purs/)
+    - backend logs can be found at `dev/logs.html`, e.g. [/dev/logs.html](http://localhost:8080/dev/logs.html). You can login here with any credentials that are also superadmins within the application.
 
 ## Quick Setup
 

